src/localization/components/locationBar.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import styles from "../locationBar.module.css"; 
+import MapComponent from './mapComponent';
+
+const LocationBar = ({ defaultLocation, onLocationChange, onRangeChange }) => {
+  const [range, setRange] = useState(30);
+  const [showMap, setShowMap] = useState(false);
+  const [locationName, setLocationName] = useState(defaultLocation);
+
+  const handleLocationClick = () => {
+    setShowMap(true);
+  };
+
+  const handleLocationSelect = (position, locationName ) => {
```

Comment:
clearly linting issue, please check your linting config 
```suggestion
  const handleLocationSelect = (position, locationName) => {
```